src/components/Messages/MessagesList.tsx

```diff
@@ -138,7 +138,12 @@ const MessagesList: FC<MessageListProps> = ({
           <InfiniteScroll
             dataLength={messages.length}
             next={fetchNextMessages}
-            style={{ display: 'flex', flexDirection: 'column-reverse' }}
+            style={{
```

Comment:
**suggestion:** can we use tailwind classes here?

Reply:
Done